doc/source/groupby.rst

```diff
@@ -66,12 +66,23 @@ pandas objects can be split on any of their axes. The abstract definition of
 grouping is to provide a mapping of labels to group names. To create a GroupBy
 object (more on what the GroupBy object is later), you may do the following:
 
-.. code-block:: python
+.. ipython:: python
+
+    df = pd.DataFrame(
+        [('bird', 'Falconiformes', 389.0),
+            ('bird', 'Psittaciformes', 24.0),
```

Comment:
this indentation doesn't seem correct, the opening parenthesis should be at the same level